cmd/plugin.go

```diff
@@ -299,6 +303,7 @@ func runPluginInstallCmd(cmd *cobra.Command, args []string) {
 func doPluginInstall(ctx context.Context, bar *uiprogress.Bar, pluginName string, wg *sync.WaitGroup, returnChannel chan *display.PluginInstallReport) {
 	var report *display.PluginInstallReport
 
+	cleanupOldTmpDirs()
```

Comment:
We don't just want to clean up old tmp dirs during plugin install. We want it for all plugin commands. The best way to achieve this is:
To call the function in the `PersistentPostRun` block in `func pluginCmd()`
```
PersistentPostRun: func(_ *cobra.Command, args []string) {
			utils.LogTime("cmd.plugin.PersistentPostRun start")
			defer utils.LogTime("cmd.plugin.PersistentPostRun end")

			cleanupOldTmpDirs()
		},
```
Add this ^^ code [here](https://github.com/turbot/steampipe/blob/9754ed0c1a84e996d820dfe75199131046d83727/cmd/plugin.go#L75)